client/components/Project/InviteComponent.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <article class="invite">
+    <h4>
+      {{ invite.name }}
+    </h4>
+    <p>Due: {{ invite.deadline }}</p>
+    <label>Collaborators: </label>
+    <ul class="reset">
+      <li
+        v-for="teammate in invite.teammates"
+        class="teammate accepted"
+      >
+        ✔️ {{ teammate }}
+      </li>
+      <li
+        v-for="teammate in invite.pendingRequests"
+        class="teammate pending"
+      >
+        🕐 {{ teammate }}
+      </li>
+    </ul>
```

Comment:
I was thinking maybe for users who are invited, this is not necessary/ too much info for them? It would be good to have this info somewhere for the project creator somewhere, such as in the project dashboard, but not for users who aren't officially in the project yet?

Reply:
Hmmm maybe we should consult the team on this, but my thoughts were that sometimes just the name might not be enough to identify the project? Do you think it'd help to at least have the email/name of the user who invited? Just in case someone did not choose a very informative name lol

Reply:
Ooh that makes sense, maybe only including the current members then? Or what you suggested seems like a good idea too! Yeah we should ask everyone though